### PR TITLE
Adding upload to S3 with Travis deploy

### PR DESCRIPTION
**Pull request by andrew-c**
#### What does this PR do? (please provide any background)

Enables deploying of the UIToolkit to an S3 bucket.

#### What tests does this PR have?

N/A

#### How can this be tested?

The first step is to get this merged. The CI should still function as normal, however when making a change to the package.json file. It should trigger a deploy (which won't actually do anything yet!).

#### What gif best describes how you feel about this work?
minon

---

- [X] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

**Comment by Joezo** [comment]
:+1: looks like it should work :)

**Comment by hx-markterry** [comment]
taken

File: scripts/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+PACKAGE_VERSION=$(cat package.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[",]//g')
```
Comment (hx-markterry) [inline scripts/deploy.sh]: good variable naming here, allows the viewer to not need to understand the shell commands

File: Gruntfile.js [hunk 2]
```diff
@@ -16,10 +16,12 @@ module.exports = function(grunt) {
   gtx.alias('docs-start', ['shell:docsStart']);
   gtx.alias('docs-view', ['shell:docsView']);
 
+  gtx.alias('upload', ['aws_s3']);
```
Comment (hx-markterry) [inline Gruntfile.js]: Can we just name this deploy? and remove line `24`

File: package.json [hunk 3]
```diff
@@ -11,14 +11,14 @@
   "scripts": {
     "build": "scripts/build.sh",
     "ci": "npm run lint && npm run test-xunit && npm run coverage",
+    "ci_travis": "npm run lint && npm run coverage",
+    "deploy": "scripts/deploy.sh",
     "postinstall": "npm run build",
     "coverage": "istanbul cover -x dist _mocha -- test/**/*-test.*",
     "predocs": "cd docs && npm install && cd - && npm run build",
     "docs": "grunt docs",
-    "pretest": "npm run build",
```
Comment (hx-markterry) [inline package.json]: I've removed these as they are no longer required before running linting or tests.

**Comment by hx-markterry** [comment]
looking good, only one tiny comment, +1 if that gets fixed.

**Pull request merged**